### PR TITLE
Tutorial step5

### PR DESCRIPTION
**Pull request by ililikakis**
**_Describe your changes here:_**

Added a short description to t8_step5_element_data.cxx and extended "How you can experiment".

**_All these boxes must be checked by the reviewers before merging the pull request:_**

- [x] The author added a BSD statement to `doc/` (or already has one)
- [x] The code compiles without warning in debugging and release mode, with and without MPI (this should be executed automatically in a github action)

  If the Pull request indroduces code that is not covered by the github action (for example coupling with a new library):
  - [ ] Should this use case be added to the github action?
  - [ ] If not, does the specific use case compile and all tests pass (check manually)

- [x] All tests pass (in various configurations, this should be executed automatically in a github action)
- [x] New source/header files are properly added to the Makefiles
- [x] The reviewer executed the new code features at least once and checked the results manually
- [x] The code is covered in an existing or new test case
- [x] The code follows the [t8code coding guidelines](https://github.com/holke/t8code/wiki/Coding-Guideline)
- [x] The code is well documented
- [x] All function declarations, structs/classes and their members have a proper doxygen documentation
- [x] All new algorithms and data structures are sufficiently optimal in terms of memory and runtime (If this should be merged, but there is still potential for optimization, create a new issue)
- [ ] Testing of this template: If you feel something is missing from this list, contact the developers

### CODE REVIEW

File: example/tutorials/t8_step5_element_data.cxx [hunk 1]
```diff
@@ -20,11 +20,26 @@
...
 */
 
-/* This is step5 of the t8code tutorials.
- * 
- * TODO: This file still needs to be documented.
+/* See also: https://github.com/holke/t8code/wiki/Step-5---Store-element-data
+ *
+ * This is step5 of the t8code tutorials.
+ * In the following we will store data in the individual elements of our forest. 
+ * To do this, we will again create a uniform forest, which will get adapt as in step4.
```
Comment (holke) [inline example/tutorials/t8_step5_element_data.cxx:27]:
```suggestion
 * To do this, we will again create a uniform forest, which will get adapted as in step4. 
```

File: example/tutorials/t8_step5_element_data.cxx [hunk 2]
```diff
@@ -20,11 +20,26 @@
...
 
-/* This is step5 of the t8code tutorials.
- * 
- * TODO: This file still needs to be documented.
+/* See also: https://github.com/holke/t8code/wiki/Step-5---Store-element-data
+ *
+ * This is step5 of the t8code tutorials.
+ * In the following we will store data in the individual elements of our forest. 
+ * To do this, we will again create a uniform forest, which will get adapt as in step4. 
+ * With the difference that we partition, balance and create ghost elements all in the same step.
```
Comment (holke) [inline example/tutorials/t8_step5_element_data.cxx:28]:
```suggestion
 * To do this, we will again create a uniform forest, which will get adapt as in step4,
 * with the difference that we partition, balance and create ghost elements all in the same step.
```

File: example/tutorials/t8_step5_element_data.cxx [hunk 3]
```diff
@@ -20,11 +20,26 @@
...
+ * After adapting the forest we will lear how to build a data array and gather data for
```
Comment (holke) [inline example/tutorials/t8_step5_element_data.cxx:29]:
```suggestion
 * After adapting the forest we will learn how to build a data array and gather data for 
```

File: example/tutorials/t8_step5_element_data.cxx [hunk 4]
```diff
@@ -20,11 +20,26 @@
...
- * 
- * TODO: This file still needs to be documented.
+/* See also: https://github.com/holke/t8code/wiki/Step-5---Store-element-data
+ *
+ * This is step5 of the t8code tutorials.
+ * In the following we will store data in the individual elements of our forest. 
+ * To do this, we will again create a uniform forest, which will get adapt as in step4. 
+ * With the difference that we partition, balance and create ghost elements all in the same step.
+ * After adapting the forest we will lear how to build a data array and gather data for 
+ * the local elements. Further we exchange the data values of the ghost elements and
```
Comment (holke) [inline example/tutorials/t8_step5_element_data.cxx:30]:
```suggestion
 * the local elements. Furthermore, we exchange the data values of the ghost elements and
```

**Review by holke — CHANGES_REQUESTED** [review]
Very good additions.
Thank you for these comments.
Only some minor typos.

I did not now about the "Hover Cells On" - thanks for that insight! Thats a cool feature :)

File: example/tutorials/t8_step5_element_data.cxx (hunk 2 above)
Comment (ililikakis) [reply example/tutorials/t8_step5_element_data.cxx]: Manually fixed.

**Pull request merged**